odl/space/ntuples.py

```diff
@@ -31,13 +31,12 @@
 from future.utils import native
 
 # External module imports
-# pylint: disable=no-name-in-module
 import ctypes
 from numbers import Integral
 from functools import partial
 from math import sqrt
 import numpy as np
-import scipy as sp
+import scipy.linalg as linalg
```

Comment:
That was the first version, which failed. The problem is the following (you need to store the script as a file and run it in a new interpreter, not in Spyder):
```
>>> import scipy as sp
>>> sp.linalg.blas.get_blas_funcs('axpy')
Traceback (most recent call last):
...
AttributeError: 'module' object has no attribute 'linalg'
```
Whereas this works:
```
>>> import scipy.linalg as linalg
>>> linalg.blas.get_blas_funcs('axpy')  # ok
```
I know it's harder to mock, but the original code triggered an error when I removed some scipy imports in the `discr_mappings.py` module.